cost/aws/unused_ip_addresses/README.md

```diff
@@ -0,0 +1,62 @@
+# AWS Unused IP Addresses
+
+## What it does
+
+This Policy finds AWS unused IP addresses and deletes them after approval.
+
+## Functional Details
+
+This policy gets a list of EIP(Elastic IPs) which can be on the following platforms:
```

Comment:
These details should describe what an unused IP address is.

Reply:
made the change